NEWS

```diff
@@ -47,23 +47,24 @@ Changes in behavior compared to prior versions:
 
 - Per Open MPI's versioning scheme (see the README), increasing the
   major version number to 3 indicates that this version is not
-  ABI-compatible with prior versions of Open MPI.  You will need to
-  recompile MPI and OpenSHMEM applications to work with this version
-  of Open MPI.
+  ABI-compatible with prior versions of Open MPI. In addition, there may
+  be differences in MCA parameter names and defaults from previous releases.
+  Command line options for mpirun and other commands may also differ from
+  previous versions. You will need to recompile MPI and OpenSHMEM applications
+  to work with this version of Open MPI.
 - With this release, Open MPI supports MPI_THREAD_MULTIPLE by default.
-- New configure option for specifying location of libnl.
-- New configure option for specifying location of zlib.
-- New configure option to request Flux PMI support.
+- New configure options have been added to specify the locations of libnl
+  and zlib.
+- A new configure option has been added to request Flux PMI support.
 - The help menu for mpirun and related commands is now context based.
   "mpirun --help compatibility" generates the help menu in the same format
   as previous releases.
 
 Removed legacy support:
 - AIX is no longer supported.
 - Loadlever is no longer supported.
-- "yoda" SPML component is removed from the Open SHMEM code base. It was utilizing
-  Open MPI BTL components for supported networks. Please see README for the currently
-  available options.
+- OpenSHMEM currently supports the UCX and MXM transports via the ucx and ikrit
```

Comment:
I think this is wrong; we should call out that we removed yoda; that's the change in customer visible behavior.

Reply:
this is what MLNX wanted.